### ISSUE TITLE
libesphttpd do not compile 

### ISSUE THREAD

**Issue by carlosmaug** [Spritetm/libesphttpd#21]
When I try to compile I get this error

CC util/captdns.c
util/captdns.c: In function 'captdnsRecv':
util/captdns.c:258:2: error: unknown type name 'remote_info'
  remote_info *remInfo=NULL;

Here is the patch to solve it

```
diff --git a/util/captdns.c b/util/captdns.c
index c7ce641..c988f7f 100644
--- a/util/captdns.c
+++ b/util/captdns.c
@@ -255,7 +255,7 @@ static void ICACHE_FLASH_ATTR captdnsRecv(struct sockaddr_in *premote_addr, char
        }
        //Send the response
 #ifndef FREERTOS
-       remote_info *remInfo=NULL;
+       remot_info *remInfo=NULL;
        //Send data to port/ip it came from, not to the ip/port we listen on.
        if (espconn_get_connection_info(conn, &remInfo, 0)==ESPCONN_OK) {
                conn->proto.udp->remote_port=remInfo->remote_port;
@@ -331,4 +331,4 @@ void ICACHE_FLASH_ATTR captdnsInit(void) {
        espconn_create(&conn);
 }

-#endif
\ No newline at end of file
+#endif
```

**Comment by ToSa27** [comment]
Same here. I just left a comment for the commit on May 27 that changed from remot_info to remote_info. Seems like @Spritetm knows more about a fix coming to the SDK but up to 1.5.4 it still needs to be remot_info. Your patch is essentially just reverting the earlier commit.

**Comment by Spritetm** [comment]
Blergh, seems they fixed the typo in 1.5.3 but reverted it in 1.5.4.

**Comment by Spritetm** [comment]
Fixed in libesphttpd f0b507656ca9a0f0c9dd92b80b4cf4e99d95a4a8 btw
